File: src/main/java/cc/polyfrost/polyweather/command/WeatherCommand.java
package cc.polyfrost.polyweather.command;

import cc.polyfrost.polyweather.PolyWeather;
import cc.polyfrost.oneconfig.utils.commands.annotations.Command;
import cc.polyfrost.oneconfig.utils.commands.annotations.Main;
import cc.polyfrost.polyweather.config.WeatherConfig;

@Command(value = PolyWeather.MODID, description = "Access the " + PolyWeather.NAME + " GUI.")
public class WeatherCommand {
    @Main
    private void main() {
        WeatherConfig.getInstance().openGui();
    }
}

File: src/main/kotlin/org/polyfrost/polyweather/command/WeatherCommand.kt
package org.polyfrost.polyweather.command

import cc.polyfrost.oneconfig.utils.commands.annotations.*
import org.polyfrost.polyweather.PolyWeather
import org.polyfrost.polyweather.config.ModConfig

@Command(value = PolyWeather.MODID, description = "Access the " + PolyWeather.NAME + " GUI.", aliases = ["weatherchanger"])
object WeatherCommand {
    @Main
    private fun main() {
        ModConfig.openGui()
    }
}